config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml

```diff
@@ -0,0 +1,154 @@
+metadata:
+  name: openstack-edpm-ipam
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+    - name: ANSIBLE_ENABLE_TASK_DEBUGGER
+      value: "True"
+  services:
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - ovn
+    - libvirt
+    - nova
+  deployStrategy:
+      deploy: true
+  baremetalSetTemplate:
+    deploymentSSHSecret: dataplane-ansible-ssh-private-key-secret
+    bmhLabelSelector:
+      app: openstack
+    ctlplaneInterface: enp1s0
+    dnsSearchDomains:
+      - osptest.openstack.org
+    cloudUserName: root
+  nodeTemplate:
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+    nodes:
+       edpm-compute-0:
+         hostName: edpm-compute-0
+         ansible:
+           ansibleUser: root
+           ansibleHost: 192.168.122.100
+           ansibleVars:
+             ctlplane_ip: 192.168.122.100
+             internal_api_ip: 172.17.0.100
+             storage_ip: 172.18.0.100
+             tenant_ip: 172.19.0.100
+             fqdn_internal_api: edpm-compute-0.example.com
+         networks:
+         - name: CtlPlane
+           subnetName: subnet1
+           defaultRoute: true
+           fixedIP: 192.168.122.100
+         - name: InternalApi
+           subnetName: subnet1
+         - name: Storage
+           subnetName: subnet1
+         - name: Tenant
+           subnetName: subnet1
+       edpm-compute-1:
+         hostName: edpm-compute-1
+         ansible:
+           ansibleUser: root
+           ansibleHost: 192.168.122.101
+           ansibleVars:
+             ctlplane_ip: 192.168.122.101
+             internal_api_ip: 172.17.0.101
+             storage_ip: 172.18.0.101
+             tenant_ip: 172.19.0.101
+             fqdn_internal_api: edpm-compute-1.example.com
+         networks:
+         - name: CtlPlane
+           subnetName: subnet1
+           defaultRoute: true
+           fixedIP: 192.168.122.101
+         - name: InternalApi
+           subnetName: subnet1
+         - name: Storage
+           subnetName: subnet1
+         - name: Tenant
+           subnetName: subnet1
+    managementNetwork: ctlplane
+    ansible:
+      ansibleUser: root
+      ansiblePort: 22
+      ansibleVars:
+         service_net_map:
+           nova_api_network: internal_api
+           nova_libvirt_network: internal_api
+         edpm_chrony_ntp_servers:
+           - clock.redhat.com
+         # edpm_network_config
+         # Default nic config template for a EDPM compute node
+         # These vars are edpm_network_config role vars
+         edpm_network_config_hide_sensitive_logs: false
+         edpm_network_config_template: templates/single_nic_vlans/single_nic_vlans.j2
+         # These vars are for the network config templates themselves and are
+         # considered EDPM network defaults.
+         neutron_physical_bridge_name: br-ex
+         neutron_public_interface_name: eth0
+         ctlplane_mtu: 1500
```

Comment:
These are IP related things are not required. Can you please use the current samples to create these?

[1] https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplane_baremetal_with_ipam.yaml